zeppelin-interpreter/src/main/java/org/apache/zeppelin/conf/ZeppelinConfiguration.java

```diff
@@ -518,7 +522,17 @@ public boolean isNotebookPublic() {
   }
 
   public String getConfDir() {
-    return getRelativeDir(ConfVars.ZEPPELIN_CONF_DIR);
+    return getRelativeDir(getString(ConfVars.ZEPPELIN_CONF_DIR));
+  }
+
+  public String getConfigFSDir() {
+    String fsConfigDir = getString(ConfVars.ZEPPELIN_CONFIG_FS_DIR);
+    if (StringUtils.isBlank(fsConfigDir)) {
+      LOG.warn(ConfVars.ZEPPELIN_CONFIG_FS_DIR.varName + " is not specified, fall back to local " +
+          "conf directory " + ConfVars.ZEPPELIN_CONF_DIR.varName);
+      return "file://" + getConfDir();
```

Comment:
shouldn't this fallback to `getString(ConfVars.ZEPPELIN_HOME) + "/" + path`? This was the behavior before this change

Reply:
Hadoop will treat it as hdfs file if no prefix with `file://`